README.md

```diff
@@ -2,7 +2,8 @@
 
 # Portfolio
 
-> A portfolio to showcase my work. 
+> A portfolio to showcase my work. Check it out at:
+https://melaniesigrid.github.io/portfolio/
```

Comment:
- [ ] [OPTIONAL] great job if you want to make your readme file look even nicer try to follow the Microverse template you can find under [this link](https://github.com/microverseinc/readme-template). You also might want to try adding a placeholder text for your link as I did in this comment by writing the following `[placeholder](link)`